posts/2018-04-10-announcing-new-2018-serverless-champions.md

```diff
@@ -0,0 +1,81 @@
+
+Well—we are back with the first round of winners for 2018!
+
+## What is a Serverless Champion?
+
+A Serverless Champion is a hero in the serverless community. They contribute to open-source serverless projects. They speak at serverless conferences and meetups. They help newcomers get excited. They crawl the forums just looking for questions to answer.
+
+**Aside:** If you, too, want to contribute to open-source, [check out our handy guide](https://serverless.com/blog/how-contribute-to-serverless-open-source/).
+
+## Announcing: our first round of Serverless Champions for 2018
+
+We conducted interviews with all three winners, and we’re including some of our favorite excerpts here. To see the interviews in their entirety (which you should!), head on over to our [Champions page](https://serverless.com/community/champions/).
+
+### Alex Casalboni
+
+<img align="left" src="https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/images/champions/2018/alex-casalboni.jpg" width="210px">
+
+Alex is the author of the [aws-lambda-power-tuning](https://github.com/alexcasalboni/aws-lambda-power-tuning) plugin. He co-organizes ServerlessDays (formerly JeffConf), frequently speaks and conferences, gives webinars, and has created many examples for the serverless community, including [this multi-region application with DynamoDB global tables](https://serverless.com/blog/build-multiregion-multimaster-application-dynamodb-global-tables/).
+
+**Q: Alex, you have been a contributor and an evangelist for the Serverless Framework for a long time. When did you start using the Serverless Framework?**
+
+AC: “I started using the Serverless Framework as soon as we realized that a system with more than a handful of Functions would never scale without proper tooling.
+
+It was May 2016 and I wrote my first review of the framework v0.5. The project was still early stage (only 60 contributors, now there are almost 400!) and the serverless ecosystem was so immature that nobody could clearly define "serverless", at least not without making a few enemies (well, that still happens, especially on Twitter!).
+
+Then I met Austen Collins at the first ServerlessConf in New York a few days later and I finally realized the scope of what the team was going to build in the upcoming months.”
+
+[Read the full interview](https://serverless.com/community/champions/alex-casalboni/), and find Alex on:
+
+[Twitter](https://twitter.com/alex_casalboni) | [GitHub](https://github.com/alexcasalboni) | [site](https://blog.alexcasalboni.com/)
+
+### Frank Schmid
+
+<img align="left" src="https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/images/champions/2018/frank-schmid.jpg" width="210px">
+
+Frank is a core maintainer of the Serverless Framework, and is incredibly proactive about stepping in on GitHub issues, and joining discussions on the forums and in Serverless Slack channels. He uses the Serverless Framework daily for his job at Stashimi. 
```

Comment:
You could also mention, that I'm author/maintainer of `serverless-webpack` 😄

Reply:
done.